pkg/services/ngalert/provisioning/alert_rules_test.go

```diff
@@ -1983,90 +1975,6 @@ func TestDeleteRuleGroups(t *testing.T) {
 	})
 }
 
-func TestIntegrationProvisiongWithFullpath(t *testing.T) {
```

Comment:
To truly have this tested, this now needs a grafana server running (so that the folder service can call to the k8s apis). Moved this test to pkg/tests/api/alerting, `TestIntegrationFullpath`.